guacamole/src/main/frontend/src/translations/de.json

```diff
@@ -399,6 +480,7 @@
         "FIELD_HEADER_IGNORE_CERT"     : "Serverzertifikat ignorieren:",
         "FIELD_HEADER_INITIAL_PROGRAM" : "Startprogramme:",
         "FIELD_HEADER_LOAD_BALANCE_INFO" : "Lastverteilungs-Info/Cookie:",
+        "FIELD_HEADER_NORMALIZE_CLIPBOARD" : "Zeilenende:",
```

Comment:
It has to do with the character on the clipboard? "Zwischenablage normalisieren / Zeilenumbruch"